ioctl/newcmd/hdwallet/hdwalletderive.go

```diff
@@ -8,9 +8,9 @@ package hdwallet
 import (
 	"fmt"
 
+	hdwallet "github.com/ethereum-optimism/go-ethereum-hdwallet"
```

Comment:
this is forked from the current "https://github.com/miguelmota/go-ethereum-hdwallet"
for safety/compatibility, better revert this change, and keep using the current repo